spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/snippet/StandardTableSnippet.java

```diff
@@ -45,11 +45,8 @@ protected StandardTableSnippet(String snippetName, Map<String, Object> attribute
     @Override
     protected Map<String, Object> createModel(Operation operation) {
         HandlerMethod handlerMethod = getHandlerMethod(operation);
-
         Collection<FieldDescriptor> fieldDescriptors = emptyList();
-        if (handlerMethod != null) {
```

Comment:
Are you sure? I remember that I added after getting a NPE here, but I don't remember what the scenario was.